src/components/Navigation/Navigation.tsx

```diff
@@ -11,13 +11,16 @@ export interface Props {
   location: string;
   sections?: SectionType[];
   children?: React.ReactNode;
+  /** @deprecated Pass a user menu into <TopBar /> instead. */
```

Comment:
Do we parse and display the deprecated doc tags anywhere?

Reply:
VSCode picks these comments up automatically, we don't parse and show them anywhere afaik but if we could do that and display them in the styleguide that would be 💯.